packages/host/app/components/operator-mode/card-schema-editor.gts

```diff
@@ -413,6 +415,10 @@ export default class CardSchemaEditor extends Component<Signature> {
   @service declare cardService: CardService;
   @service declare operatorModeStateService: OperatorModeStateService;
 
+  private writeTask = restartableTask(async (src: string) => {
+    this.args.file.write(src, true);
```

Comment:
does this need a `return` or an `await`? I assume it is a task because we care about the async?

Reply:
yeah, and in fact we might leak async in our tests if we don't await...

Reply:
actually based on the addField PR discussion--i think however you handle that there, you just handle this in the same manner